docs/source/tracking.rst

```diff
@@ -150,6 +154,53 @@ just one block of code as follows:
 The run remains open throughout the ``with`` statement, and is automatically closed when the
 statement exits, even if it exits due to an exception.
 
+
+Performance Tracking with Metrics
+---------------------------------
+
+You log MLflow metrics with ``log`` methods in the Tracking API. The ``log`` methods support two alternative methods for distinguishing metric values on the x-axis: ``timestamp`` and ``step``. 
+
+``timestamp`` is an optional long value that represents the time that the metric was logged. ``timestamp`` defaults to the current time. ``step`` is an optional integer that represents any measurement of training progress (number of training iterations, number of epochs, and so on). ``step`` defaults to 0 and has the following requirements and properties:
+
+- Must be a valid 64-bit integer value.
+- Can be negative.
+- Can be out of order in successive write calls. For example, (1, 3, 2) is a valid sequence.
+- Can have "gaps" in the sequence of values specified in successive write calls. For example, (1, 5, 75, -20) is a valid sequence.
+
+If you specify both a timestamp and a step, metrics are recorded against both axes independently.
+
+Examples
+~~~~~~~~
+
+Java and Scala
+  .. code-block:: scala
+  
+    logMetric(String runUuid, String key, double value, long timestamp, int step)
```

Comment:
These aren't really examples so much as the API definition. Do we want examples instead?